### ISSUE TITLE
Bug when calling `./craft resave/navigation-nodes`

### ISSUE THREAD

**Issue by rungta** [verbb/navigation#362]
### Describe the bug

```
$ ./craft resave/navigation-nodes
Exception 'TypeError' with message 'craft\console\controllers\ResaveController::saveElements():
Argument #1 ($query) must be of type craft\elements\db\ElementQueryInterface,
string given, called in .../vendor/verbb/navigation/src/Navigation.php on line 312'

in .../vendor/craftcms/cms/src/console/controllers/ResaveController.php:433
```

### Steps to reproduce

1.  Run the ` ./craft resave/navigation-nodes` console command via the command line

### Craft CMS version

4.4.15

### Plugin version

2.0.19

### Multi-site?

Yes

**Comment by engram-design** [comment]
Fixed in [2.0.20](https://github.com/verbb/navigation/releases/tag/2.0.20)

**Comment by rungta** [comment]
Thanks Josh!
